src/sphinxcontrib/towncrier/__init__.py

```diff
@@ -85,8 +85,8 @@ def _get_draft_version_fallback(strategy: str, sphinx_config: Dict[str, Any]):
     return '[UNRELEASED DRAFT]'
 
 
-def _nodes_from_rst(state: statemachine.State, rst_source: str) -> List[nodes.Node]:
-    """Turn an RST string into a list of nodes that can be used in the document."""
+def _nodes_from_rst(state: statemachine.State, rst_source: str) -> List[nodes.Node]:  # pylint: disable=line-too-long
+    """Turn an RST string into a list of nodes that can be used in the document."""  # pylint: disable=line-too-long
```

Comment:
This mustn't be ignored but should be rather re-phrased. Docstring title line should fit in 72 chars including the quotes per PEP257. Another linter, pydocstyle, will probably yell at this too.

Reply:
Oh, it's PEP8 that suggests the 72 char limit: https://www.python.org/dev/peps/pep-0008/#maximum-line-length.